src/Outline/TemplateBuilder.php

```diff
@@ -153,4 +153,18 @@ private function close() {
 		return "}}";
 	}
 
+	function getIntroTemplate(): string {
+		if ( $this->params['introtemplate'] ) {
+			return $this->open( $this->params['introtemplate'] ) . $this->close();
+		}
+		return "";
+	}
+
+	function getOutroTemplate(): string {
+		if ( $this->params['outrotemplate'] ) {
```

Comment:
Oops, I should have been more clear. With dropping the negation I meant changing `!==` to `===`. Explicit checks are definitely better than implicit checks. Right now if you have a template named `0` the check will fail.

Reply:
https://github.com/SemanticMediaWiki/SemanticResultFormats/pull/585